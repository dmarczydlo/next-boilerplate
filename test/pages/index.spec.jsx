import React from 'react';
import Index from '../../pages/index';

describe('Index site', () => {
    let wrapper = null;

    it('should be rendered', () => {
        wrapper = global.shallow(<Index/>);
        expect(wrapper).toMatchSnapshot();
    });
});
