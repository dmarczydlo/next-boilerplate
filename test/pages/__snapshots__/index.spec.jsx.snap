// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index site should be rendered 1`] = `
<div>
  <h1>
    Title nextjs boilerplate
  </h1>
</div>
`;
